src/rules/color-no-named/__tests__/index.js

```diff
@@ -0,0 +1,53 @@
+import {
+  ruleTester,
+  warningFreeBasics,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule(undefined, tr => {
+  warningFreeBasics(tr)
+
+  tr.ok("a { color: #000; }")
+  tr.ok("a { color: #ababab; }")
+  tr.ok("a { color: rgba(0, 0, 0, 0); }")
+  tr.ok("a { something: #000, #fff, #333; }")
+
```

Comment:
Please add tests ensuring that color names are ignored if they are within comments, strings, or urls without quotes.